cmd/kube-controller-manager/app/plugins.go

```diff
@@ -123,9 +122,6 @@ func ProbeControllerVolumePlugins(logger klog.Logger, config persistentvolumecon
 		return allPlugins, err
 	}
 
-	allPlugins = append(allPlugins, local.ProbeVolumePlugins()...)
```

Comment:
It would be great if you renamed the function to `ProbeProvisionableControllerVolumePlugins`  to make clear what it is, like we have `ProbeControllerVolumePlugins` and `ProbeExpandableVolumePlugins`. 

The rest of the code lgtm.

Reply:
`ProbeProvisionableControllerVolumePlugins` is not suitable. The returned plugins include provisioner, recycler and deleter. i.e. the nfs plugin doesn't implement the provisioner and deleter, but it is still returned.

If https://github.com/kubernetes/enhancements/issues/4547 is implemented, the controller will only return the plugins that implement the recycler interface. So, `ProbeRecyclableControllerVolumePlugins` or `ProbeLegacyRecyclableControllerVolumePlugins` may be a better name once the enhancement https://github.com/kubernetes/enhancements/issues/4547 is implemented.

Let's keep the current name for now, and update it later. What do you think? @jsafrane

Reply:
Ack